### DISCUSSION TITLE
Requests not showing up in wallets

### DISCUSSION THREAD

**Discussion by MhAhmadAli** [WalletConnect/walletconnect-monorepo#3858] [Developer Support]
Hi! I am working on an android dapp that uses the Wallet Connect Kotlin SDK.

The issue I am facing is that when connected to a wallet lets say metamask. When I send any request (lets assume its `personal_sign`), I am switched to metamask, I insert my password but don't see any request popup (I have tried waiting upto 10 mins).

If I keep metamask unlocked in background and send a request, the popup shows up instantly.

**Note:** this issue occurs with every wallet app I have tested (metamask, trust wallet, coinbase).

I have the `expiry` parameter set to `now() + MIN_INTERVAL`, just like it said in the docs. I have tried without setting it as well.

Am I missing some config option??

**Comment by glitch-txs** [comment]
hi, could you open this in a GitHub issue in the Kotlin repository? Thanks!
